modules/src/ics03_connection/msgs/conn_open_ack.rs

```diff
@@ -55,7 +56,7 @@ impl MsgConnectionOpenAck {
     /// value `Height(0)` if this field is not set.
     pub fn consensus_height(&self) -> Height {
         match self.proofs.consensus_proof() {
-            None => Height::from(0_u32),
+            None => zero_height(),
```

Comment:
Just an aesthetic idea: This would look nicer in my opinion if we typed it as `Height::zero()`. The current method `zero_height()` is an improvement over the older one (`Height::from(0_u32)`) but is somehow odd, as if the method was local to this module..

Reply:
44490c4